contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,202 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+Author(s): [Ronak Lakhotia](https://github.com/RonakLakhotia)
+
+## What Is Angular?
+
+Angular is an open-source frontend Javascript framework that helps you build <tooltip content="Web apps that are more responsive to user interactions ">Single Page Web Applications</tooltip>.
+The framework is designed to better engineer applications in an effort to make development and testing easier for enterprise web applications. Angular is maintained by Google, as well as a community of individual developers.
```

Comment:
This sentence feels a bit verbose. Something like this? `The framework aims to make development and testing of enterprise web applications easier.`?